.github/workflows/ci.yaml

```
@@ -111,13 +111,16 @@ jobs:
       - uses: actions/checkout@v4
       - name: "Install Rust toolchain"
         run: rustup show
-      - name: "Install cargo insta"
+      - name: "Install mold"
+        uses: rui314/setup-mold@v1
+      - name: "Install cargo nextest"
         uses: taiki-e/install-action@v2
         with:
-          tool: cargo-insta
+          tool: cargo-nextest
       - uses: Swatinem/rust-cache@v2
       - name: "Run tests"
-        run: cargo insta test --all --all-features --unreferenced reject
+        shell: bash
+        run: cargo nextest run --all --status-level skip --failure-output immediate-final --no-fail-fast -j 12
```

`--all` is deprecated

```suggestion
        run: cargo nextest run --workspace --status-level skip --failure-output immediate-final --no-fail-fast -j 12
```